controllers/helmchart_controller.go

```diff
@@ -527,7 +527,15 @@ func (r *HelmChartReconciler) buildFromHelmRepository(ctx context.Context, obj *
 
 		if file != "" {
 			defer func() {
-				os.Remove(file)
+				if err := os.Remove(file); err != nil {
+					r.eventLogf(
+						ctx,
+						obj,
+						corev1.EventTypeWarning,
+						meta.FailedReason,
+						"failed to delete temporary credentials file: %s",
+						err)
```

Comment:
We already have a style for multi-line usage of this function. Something like:
```
r.eventLogf(ctx, obj, events.EventTypeTrace, "NoSourceArtifact",
    "no artifact available for %s source '%s'", obj.Spec.SourceRef.Kind, obj.Spec.SourceRef.Name)
```
Let's keep it consistent.